conformance/conformance_test.cc

```diff
@@ -270,8 +292,20 @@ void ConformanceTestSuite::VerifyResponse(
   string differences;
   differencer.ReportDifferencesToString(&differences);
 
-  bool check;
-  check = differencer.Compare(*reference_message, *test_message);
+  bool check = false;
+
+  if (require_same_wire_format) {
+    GOOGLE_DCHECK_EQ(response.result_case(),
+                    ConformanceResponse::kProtobufPayload);
+    const string& protobuf_payload =  response.protobuf_payload();
+    check = (equivalent_wire_format.compare(
```

Comment:
I think instead of doing `equivalent_wire_format.compare(...) == 0` you can just do `equivalent_wire_format == protobuf_payload`.